### PR TITLE
Added category to blog detail page

### LINKED ISSUE #3748: Blog detail page: add category
Add category link at the top of the blog (after the by line and the date).

image

### PR DESCRIPTION
**Pull request by youriwims**
Closes #3748

Test Page: https://foundation-mofostaging-pr-3812.herokuapp.com/en/blog/different-recognize-perhaps-clearly-push-today/

### CODE REVIEW

**Comment by kristinashu** [comment]
It's showing up and working well! But could you please add the purple/blue hover state that categories have on /blog?

**Comment by youriwims** [comment]
Ah, yes—not a problem. Should be there now!

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html [hunk 1]
```diff
@@ -13,7 +13,15 @@
   <div class="py-4 py-md-5 col-lg-8">
     <div class="cms">
       <h1 class="h1-heading">{{ page.title }}</h1>
-      <p class="h6-heading mb-5">By {{ page.author }} | {% if page.first_published_at %}{{ page.first_published_at|date:"F j, Y" }}{% else %}not published yet{% endif %}</p>
+      <p class="h6-heading mb-5">
+        By {{ page.author }} | {% if page.first_published_at %}{{ page.first_published_at|date:"F j, Y" }}{% else %}not
+        published yet{% endif %} |
+        {% with category=page.specific.category.first %}
+        {% if category %}
+        <a class="blog-detailed-category" href="/blog/category/{{category.slug}}">{{ category }}</a>
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html]: we probably shouldn't be hardcoding parts of this URL, we should be able to ask wagtail for the URL we need instead.

Comment (Pomax) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html]:
```suggestion
        <a class="blog-detailed-category" href="{% slugurl 'blog' %}category/{{category.slug}}">{{ category }}</a>
```

Comment (Pomax) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html]: ideally we also don't hardcode `category` but fetch that from the BlogPageCategory model (e.g using a `fullUrl` property definition) but that's beyond the scope of this issue, really.

File: source/sass/views/blog.scss [hunk 2]
```diff
@@ -17,3 +17,9 @@
     }
   }
 }
+
+a.blog-detailed-category {
```
Comment (Pomax) [inline source/sass/views/blog.scss]: Do we need "blog-detailed" here? Can call this just `category` without running into conflicts?

Comment (youriwims) [reply source/sass/views/blog.scss]: Looks like that's completely possible—updating now.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html (hunk 1 above)
Comment (youriwims) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html]: @Pomax Definitely, not a problem. In `featured_blogs.html` the blog category url is hardcoded similar to what I've done here, just fyi for future fixes/updates 👍 .

**Pull request merged**